app/util/cmr.ts

```diff
@@ -91,6 +107,8 @@ export interface CmrQuery
   concept_id?: string | string[];
   page_size?: number;
   downloadable?: boolean;
+  user_id?: string;
```

Comment:
These seem specific to an ACL Query - I think I'd add another interface that extends CmrQuery called something like CmrAclQuery that adds those fields and is only used in the ACL query.

Reply:
Good point! Updated here: https://github.com/nasa/harmony/commit/2c80857ead78e60e45c7425bdb946ae9f98213e8